joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -1120,6 +1120,27 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
     callNode.columnNumber shouldBe Some(9)
   }
 
+  "method defined inside a class using << operator" in {
```

Comment:
Wasn't this test case already added [here](https://github.com/joernio/joern/pull/3367/files#diff-45b026c78f0405d131600a5e173a4dc262cdaae9ad115dfae83deb1ac6f1a938R1123)?

Reply:
yes, don't know why commits are already part of master branch are still showing here, investigating.